<launch>
  <param name="/initial_side" value="-1"/>
  <param name="direction" value="1"/>
  <param name="current_node" value="1"/>
  <param name="/initial_speed" value="23"/>  
  
  <node name="talker" pkg="race" type="talker.py"/>
  
  <node name="teensy" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0"/>
  
  <node name="lidar" pkg="urg_node" type="urg_node" args="_ip_address:='192.168.1.11'"/>  
 
  <node name="dist_finder" pkg="race" type="dist_finder.py"/> 

  <node name="control" pkg="race" type="control.py"/>

  <node name="goFastOrGoHome" pkg="race" type="goFastOrGoHome.py" output="screen" />

</launch>
